api/src/main/resources/org/openmrs/api/db/hibernate/CohortMembership.hbm.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+    <class name="CohortMembership" table="cohort_member">
+        <id name="cohortMemberId" type="int" column="cohort_member_id">
+            <generator class="native">
+                <param name="sequence">cohort_member_cohort_member_id_seq</param>
+            </generator>
+        </id>
+
+        <many-to-one name="cohort" column="cohort_id" class="Cohort" access="field"/>
```

Comment:
You need to make this not nullable, it will give you a clue if the cohort field is getting saved or not